slack/commands.py

```diff
@@ -287,6 +287,13 @@ def command_slack_reply(buffer: str, args: List[str], options: Options):
         run_async(slack_buffer.post_message(split_args[1], thread_ts, broadcast))
 
 
+@weechat_command("away|active")
+def command_slack_presence(buffer: str, args: List[str], options: Options):
+    slack_buffer = shared.buffers.get(buffer)
+    if slack_buffer:
+        run_async(slack_buffer.workspace.api.set_presence(args[0]))
```

Comment:
You have to check that `args[0]` is either `active` or `away`, and otherwise print an error.

The error message should be in this format (because that's what WeeChat's built in commands print when you give an unrecognized argument):

```
Error with command "/slack presence asd" (help on command: /help slack presence)
```